File: coco/src/main/java/coco/ep/m7i2/finalized/Power.java
package coco.ep.m7i2.finalized;

public class Power extends Exp implements coco.ep.m7i2.Power<Exp> {

    protected Exp left;
    protected Exp right;

    public Power(Exp _left, Exp _right) {
        this.left = _left;
        this.right = _right;
    }

    public Exp getLeft() {
        return left;
    }
    public Exp getRight() {
        return right;
    }   
    
    public void setLeft(coco.ep.Exp<Exp> _left) {
        left = convert(_left);
    }
    public void setRight(coco.ep.Exp<Exp> _right) {
        right = convert(_right);
    }
}

File: coco_csharp/CoCo/EP/M7I2/Finalized/Power.cs
namespace EP.M7I2.Finalized {
    public class Power : Exp, EP.M7I2.Power<Exp> {
        private EP.Exp<Exp> left;
        private EP.Exp<Exp> right;

        public Power(EP.Exp<Exp> _left, EP.Exp<Exp> _right) {
            this.left = _left;
            this.right = _right;
        }

        public EP.Exp<Exp> getLeft() { return this.left; }
        public void setLeft(EP.Exp<Exp> _left) { this.left = _left; }

        public EP.Exp<Exp> getRight() { return this.right; }
        public void setRight(EP.Exp<Exp> _right) { this.right = _right; }
    }
}